lib/ui/viewer/gallery/gallery_app_bar_widget.dart

```diff
@@ -19,15 +20,15 @@ class GalleryAppBarWidget extends StatefulWidget {
   final GalleryType type;
   final String title;
   final SelectedFiles selectedFiles;
-  final String path;
+  final DevicePathCollection devicePathCollection;
```

Comment:
Weird variable name? o_O

Reply:
I didn't want to continue with DeviceFolder to avoid confusion between iOS Folder, iOS Album, and Android Folder. 
I will document that this entity represents Device Folder on Android & iOS Album (Auto-Created + User Created) on iOS.
We can rename it to  `DeviceCollection` or something else? 
FYI: The `DevicePathCollection` was inspired from `Device Folder`, `AssetPathEntity` and `Collection` 🤣

Reply:
`DeviceCollection` sounds the best